web/src/components/Footer.tsx

```diff
@@ -16,11 +16,13 @@ const FooterContainer = styled('footer')`
   gap: ${props => props.theme.spacing(2)};
   background-color: ${props => props.theme.legacy.colors.backgroundAccentColor};
   padding-bottom: ${props => props.theme.spacing(2)};
+  color: ${props => props.theme.palette.text.primary};
 
   ${props => props.theme.breakpoints.up('md')} {
     background-color: ${props =>
       props.theme.isContrastTheme ? props.theme.palette.secondary.dark : props.theme.palette.secondary.light};
-    padding: 0 4px;
+    padding: 8px;
+    color: inherit;
```

Comment:
🤔 Is this actually necessary? I don't see a difference

Reply:
I can see a difference... the text color got more contrast while contrast mode is enabled (without it its off white)